cdap-docs/developers-manual/source/building-blocks/workflows.rst

```diff
@@ -31,6 +31,15 @@ single start node and a single finish node. As described below, execution can be
 sequential (the default) or :ref:`concurrent <workflow-concurrent>`, and the graph can be 
 either a simple series of nodes or a more complicated :ref:`parallel workflow <workflow_parallel>`.
 
+Workflows can be controlled by the :ref:`CDAP CLI <cli>` and the :ref:`Lifecycle HTTP
+RESTful API <http-restful-api-lifecycle>`. The :ref:`status of a workflow
+<http-restful-api-lifecycle-start-stop-status>` can be retrieved, and individual workflows
```

Comment:
Workflow are programs so they have start and stop just like other programs. Now a workflow can have one or more than one schedule associated to it.

These schedules are SUSPENDED state when the  app is deployed for the first time and each and every schedule needs to be resumed changing its status to SCHEDULED as required by the user.